src/Analyzers/Core/Analyzers/EnforceOnBuildValues.cs

```diff
@@ -96,6 +96,7 @@ internal static class EnforceOnBuildValues
     public const EnforceOnBuild UseCollectionExpressionForCreate = /*IDE0303*/ EnforceOnBuild.Recommended;
     public const EnforceOnBuild UseCollectionExpressionForBuilder = /*IDE0304*/ EnforceOnBuild.Recommended;
     public const EnforceOnBuild UseCollectionExpressionForFluent = /*IDE0305*/ EnforceOnBuild.Recommended;
+    public const EnforceOnBuild MakeAnonymousFunctionStatic = /*IDE0310*/ EnforceOnBuild.Recommended;
```

Comment:
start at 320.  i want to leave 300-319 as collection-expr space.

Reply:
So many for collection expressions?.. Ok, as you wish)

Reply:
Yes. There are more patterns to detect, and we're adding dict and natural type support. These will likely come with several analyzers.